source/Calamari.Tests/Fixtures/Integration/Packages/DockerImagePackageDownloaderFixture.cs

```diff
@@ -114,7 +114,7 @@ public void PackageWithWrongCredentials_FailsWithRetry()
             var exception = Assert.Throws<CommandException>(() => downloader.DownloadPackage("octopus-echo",
                 new SemanticVersion("1.1"), "docker-feed",
                 new Uri(AuthFeedUri),
-                FeedUsername, "SuperDooper",
+                "Nonexistantuser", "SuperDooper",
```

Comment:
To fix an issue with a flakey test
https://octopusdeploy.slack.com/archives/C01HH8T16G3/p1728877229231419